plexpy/__init__.py

```diff
@@ -421,6 +430,22 @@ def dbcheck():
         'keep_history INTEGER DEFAULT 1, deleted_user INTEGER DEFAULT 0)'
     )
 
+    # notify_log table :: This is a table which logs notifications sent
+    c_db.execute(
+        'CREATE TABLE IF NOT EXISTS notify_log (id INTEGER PRIMARY KEY AUTOINCREMENT, '
+        'session_key INTEGER, rating_key INTEGER, user_id INTEGER, user TEXT, '
+        'agent_id INTEGER, agent_name TEXT, on_play INTEGER, on_stop INTEGER, on_watched INTEGER, '
+        'on_pause INTEGER, on_resume INTEGER, on_buffer INTEGER)'
+    )
+
+    # library_sections table :: This table keeps record of the servers library sections
```

Comment:
You should log the server id here too. Will make things easier if you change the server in the settings. It should also update this table every time you switch to a new server.